mods/taskbar-autohide-better.wh.cpp

```diff
@@ -0,0 +1,98 @@
+// ==WindhawkMod==
+// @id              taskbar-autohide-better
+// @name            Better Taskbar Autohide
+// @description     Allow taskbar autohide when inactive window notified
+// @version         1.0
+// @author          Cirn09
+// @github          https://github.com/Cirn09
+// @twitter         https://twitter.com/jack
+// @homepage        https://blog.cirn09.xyz/
+// @include         explorer.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Better Taskbar Autohide
+Allow taskbar autohide when inactive window notified
```

Comment:
Up to you, but I think the description can be improved. What does "inactive window notified" mean? What happens without the mod and what changes when the mod is used?